### PR TITLE
Update alert-manager.rst

### PR DESCRIPTION
**Pull request by aantn**
I've tried to make the instructions more clear.

@pavangudiwada and @arikalon1 can you both review? Do **not** merge yet. I'd like feedback on:

1. Are the instructions clear and consistent?
2. Do you know which instructions to follow and when?
3. Do we have links elsewhere (in the UI or in Slack messages) that link to specific sections of this page? Do we need to update them?

### CODE REVIEW

**Comment by aantn** [comment]
For your convenience, here are links to the new and old version:

* New (this PR): https://docs.robusta.dev/improve-alert-manager-docs/user-guide/alert-manager.html
* Old: https://docs.robusta.dev/master/user-guide/alert-manager.html

**Comment by aantn** [comment]
TODO: this change needs to be ported to the new docs too.

**Review by arikalon1 — APPROVED** [review]
Looks good.
Left some comments.

Will these references work?
configuration.rst line 25,26
architecture.rst line 27

File: docs/user-guide/alert-manager.rst [hunk 1]
```diff
@@ -114,13 +107,44 @@ If AlertManager is located outside of your Kubernetes cluster then a few more st
...
+      grafana_url: ""
+      prometheus_url: "http://PROMETHEUS_SERVICE_NAME.monitoring.svc.cluster.local:9090"
 
 
 Alerts silencing
------------------------------------------
+=================
 
-Robusta enables silencing AlertManager alerts directly from your notification channels (Sinks)
+Robusta enables silencing AlertManager alerts directly from your notification channels (sinks).
```
[the remark this answers is not in the document]
Comment (pavangudiwada) [reply docs/user-guide/alert-manager.rst:147]: Agreed, it's clear that Robusta only identifies AlertManagers in the same cluster. Without something specific about non-grafana AlertManagers, my assumption is "alertmanager_url:" is sufficient.

File: docs/user-guide/alert-manager.rst [hunk 2]
```diff
@@ -114,13 +107,44 @@ If AlertManager is located outside of your Kubernetes cluster then a few more st
             repeat_interval: 4h
             continue: true
 
-The `TOKEN` format is: `ACCOUNT_ID SIGNING_KEY`
+Robusta's Embedded Prometheus
+-----------------------------
+If you installed Robusta's :ref:`Embedded Prometheus Stack` then no configuration is necessary.
```
[the remark this answers is not in the document]
Comment (aantn) [reply docs/user-guide/alert-manager.rst:112]: Yes, I think it's fine to repeat here.

File: docs/user-guide/alert-manager.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (aantn) [reply docs/user-guide/alert-manager.rst:147]: Fixed.

File: docs/user-guide/alert-manager.rst [hunk 3]
```diff
@@ -34,55 +36,45 @@ Add this snippet to your Prometheus AlertManager **values.yaml** file.
...
-
-      The ``default`` in the webhook_config url, is the namespace robusta is installed on. If you installed Robusta on a different namespace, update the url accordingly.
-
-    .. note::
-
-      When a ``route`` is matched, the alert will not be sent to the following routes, unless the ``route`` is configured with ``continue: true``
-      Make sure the Robusta ``route`` is the first ``route`` defined.
+.. admonition:: Common Mistakes
 
+    1. The ``default`` in the webhook_config url, is the namespace robusta is installed on. If you installed Robusta on a different namespace, update the url accordingly.
```
[the remark this answers is not in the document]
Comment (aantn) [reply docs/user-guide/alert-manager.rst:41]: Yes, added.

File: docs/user-guide/alert-manager.rst [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 
-.. admonition:: Robusta receiver and route
+.. admonition:: AlertManager config for sending alerts to Robusta
 
     .. code-block:: yaml
 
         receivers:
           - name: 'robusta'
             webhook_configs:
+              # the following line assumes that Robusta was installed in the `default` namespace.
```
[the remark this answers is not in the document]
Comment (aantn) [reply docs/user-guide/alert-manager.rst:26]: Made it less redundant.

File: docs/user-guide/alert-manager.rst [hunk 5]
```diff
@@ -1,31 +1,33 @@
...
 If you installed Robusta's :ref:`Embedded Prometheus Stack` then no configuration is necessary.
+For all other setups, read on!
 
-AlertManager receiver and route definition
---------------------------------------------
-
-Add this snippet to your Prometheus AlertManager **values.yaml** file. 
+General Instructions
+======================
+For Prometheus to send alerts to Robusta, add the following to AlertManager's config file.
```
[the remark this answers is not in the document]
Comment (aantn) [reply docs/user-guide/alert-manager.rst:16]: Done.

**Comment by aantn** [comment]
> Will these references work?
Yes, they all work. They either link to titles that still exist or link to the page by filename not title. I double checked.

**Pull request merged**